docs/make.jl

```diff
@@ -32,7 +32,8 @@ pages = OrderedDict(
             "...improve performance with time series data" => "how_to/improve_ts_performance.md",
             "...write and read data with a JSON" => "how_to/serialize_data.md",
             "...reduce REPL printing" => "how_to/reduce_repl_printing.md",
-            "...migrate from version 3.0 to 4.0" => "how_to/migrating_to_psy4.md"
+            "...migrate from version 3.0 to 4.0" => "how_to/migrating_to_psy4.md",
+            "...build system using csv files" => "how_to/build_system_with_files.md"
```

Comment:
```suggestion
            "...build a `System` using .csv files" => "how_to/build_system_with_files.md"
```
Let's also move this up higher, above the current "...parse tabular data from .csv files".

Reply:
Also, there is currently a warning in [Parse Tabular Data from .csv Files](https://nrel-sienna.github.io/PowerSystems.jl/stable/how_to/parse_tabular_data/#table_data) about deprecating the table data Parser -- let's add a reference to this how-to in the last line of that warning